ci/ray_ci/container.py

```diff
@@ -96,7 +96,7 @@ def _get_run_command(
         command += [
             "--workdir",
             "/rayci",
-            "--shm-size=2.5gb",
+            "--shm-size=4.77gb",
```

Comment:
is this the right number? we have different memory size on different machines. the smallest one, m7i, has 8gb of memory.

can we make this dependent of the memory size of the machine?

that particular one is `m7i.xlarge` in the example, which has 16GB of memory.

what is the criteria to use tmp or shm? can that behavior be forced? why unit tests need large shm?

Reply:
also, will this prevent tests to run in parallel?

Reply:
large machine has 32gb, medium machine has 16 gb, each has 2-3 parallelization max so it has spaces for 4.77 each

small machine has 8gb so it can oom if those tests go crazy, we'll see how it behaves in the test run

Reply:
ok.. anyways.. the constant just feels a bit ad-hoc, and a source of future flakiness. like if this is calculated based on the parallelism and the total memory size, I will feel more comfortable. 

also feel that there should be an option to force ray to use shm rather than tmp. tmp in container is actually backed by disks, not memory.

maybe at least add a comment on where this number comes from and why it is required? I would also round it up to 5gb maybe.